script/50_CoreAndPeriphery.s.sol

```diff
@@ -2,49 +2,139 @@
 
 pragma solidity ^0.8.0;
 
-import {ScriptUtils} from "./utils/ScriptUtils.s.sol";
+import {BatchBuilder, Vm, console} from "./utils/ScriptUtils.s.sol";
+import {ERC20BurnableMintableDeployer, RewardTokenDeployer} from "./00_ERC20.s.sol";
 import {Integrations} from "./01_Integrations.s.sol";
 import {PeripheryFactories} from "./02_PeripheryFactories.s.sol";
 import {EVaultImplementation} from "./05_EVaultImplementation.s.sol";
 import {EVaultFactory} from "./06_EVaultFactory.s.sol";
-import {Lenses} from "./08_Lenses.s.sol";
-import {EVKPerspectives} from "./09_Perspectives.s.sol";
+import {
+    LensAccountDeployer,
+    LensOracleDeployer,
+    LensIRMDeployer,
+    LensVaultDeployer,
+    LensUtilsDeployer,
+    LensEulerEarnVaultDeployer
+} from "./08_Lenses.s.sol";
+import {
+    EVKFactoryPerspectiveDeployer,
+    PerspectiveGovernedDeployer,
+    EVKPerspectiveEscrowedCollateralDeployer,
+    EVKPerspectiveEulerUngoverned0xDeployer,
+    EVKPerspectiveEulerUngovernedNzxDeployer
+} from "./09_Perspectives.s.sol";
 import {Swap} from "./10_Swap.s.sol";
 import {FeeFlow} from "./11_FeeFlow.s.sol";
-import {EVaultFactoryGovernorDeployer} from "./12_Governor.s.sol";
+import {
+    EVaultFactoryGovernorDeployer,
+    TimelockControllerDeployer,
+    GovernorAccessControlEmergencyDeployer
+} from "./12_Governor.s.sol";
 import {TermsOfUseSignerDeployer} from "./13_TermsOfUseSigner.s.sol";
+import {NttManagerDeployer, WormholeTransceiverDeployer} from "./14_NTT.s.sol";
+import {FactoryGovernor} from "./../src/Governor/FactoryGovernor.sol";
+import {GovernorAccessControlEmergency} from "./../src/Governor/GovernorAccessControlEmergency.sol";
+import {ERC20BurnableMintable} from "./../src/ERC20/deployed/ERC20BurnableMintable.sol";
+import {RewardToken} from "./../src/ERC20/deployed/RewardToken.sol";
 import {Base} from "evk/EVault/shared/Base.sol";
 import {ProtocolConfig} from "evk/ProtocolConfig/ProtocolConfig.sol";
+import {AccessControl} from "openzeppelin-contracts/access/AccessControl.sol";
+import {TimelockController} from "openzeppelin-contracts/governance/TimelockController.sol";
+import {NttManager} from "native-token-transfers/NttManager/NttManager.sol";
+import {WormholeTransceiver} from "native-token-transfers/Transceiver/WormholeTransceiver/WormholeTransceiver.sol";
+
+contract CoreAndPeriphery is BatchBuilder {
+    uint256 internal constant HUB_CHAIN_ID = 1;
+    address internal constant BURN_ADDRESS = 0x000000000000000000000000000000000000dEaD;
+    uint8 internal constant EUL_DECIMALS = 18;
+    address internal constant EVAULT_FACTORY_GOVERNOR_PAUSER = 0xff217004BdD3A6A592162380dc0E6BbF143291eB;
```

Comment:
Let's label this as Hypernative's address in variable name and the script output, just to be clear.